frameworks/g-cloud-9/questions/services/standardsCSASTARWhen.yml

```diff
@@ -0,0 +1,17 @@
+name: CSA STAR accreditation date
```

Comment:
For dates in the briefs stuff we give a hint `hint: 'eg 31/12/2020'` - don't know if that would help here?